### ISSUE TITLE
[BUG] Only first variable in JSON body is being substituted

### ISSUE THREAD

**Issue by roopemerikukka** [usebruno/bruno#1607]
I have a request with JSON body like this:

```json
{
  "username": "{{username}}",
  "password": "{{password}}"
}
```

But when inspecting timeline what is actually sent I can see that the password in the payload is empty.

```
> POST https://myendpoint.com/login
> Content-Type: application/json
> data {"username":"my-username","password":""}
```

And if I switch the fields in other way around like this the username is missing:

```json
{
  "password": "{{password}}",
  "username": "{{username}}"
}
```

```
> POST https://myendpoint.com/login
> Content-Type: application/json
> data {"password":"my-password","username":""}
```

So it seems like that only the first variable is being substituted.

I'm using version v1.6.1.

**Comment by sanjai0py** [comment]
Hey @roopemerikukka, the bug you reported has been fixed in version `v1.9.0`. Could you please try using that version?

**Comment by roopemerikukka** [comment]
Oh yeah works now, thanks for pointing out that I had outdated client :)
